estate_account/models/estate_property.py

```diff
@@ -0,0 +1,21 @@
+from odoo import Command, models, _
+
+
+class EstateProperty(models.Model):
+    _inherit = "estate.property"
+
+    def action_set_sold(self) -> bool:
+        percent_description = _("6 percent of selling price")
+        admin_fee_description = _("Administrative fees")
```

Comment:
You don't need to explicitly call `_()` for field values. If the fields are marked as translatable, the system takes care of the rest.